spec/wizards/a_level_steps/are_any_a_levels_required_for_this_course_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ALevelSteps::AreAnyALevelsRequiredForThisCourse do
+  subject(:wizard_step) { described_class.new }
+
+  describe 'validations' do
+    it 'is valid with a valid answer' do
+      wizard_step.answer = 'yes'
+      expect(wizard_step).to be_valid
+    end
+
+    it 'is not valid without an answer' do
+      wizard_step.answer = nil
+      expect(wizard_step).not_to be_valid
+      expect(wizard_step.errors.added?(:answer, :blank)).to be true
+    end
+  end
+
+  describe '.permitted_params' do
+    it 'returns the permitted params' do
+      expect(described_class.permitted_params).to eq([:answer])
+    end
+  end
+
+  describe '#previous_step' do
+    it 'returns the symbol for the previous step' do
+      expect(wizard_step.previous_step).to eq(:first_step)
+    end
+  end
+
+  describe '#next_step' do
+    context 'when a level is required' do
+      before do
+        wizard_step.answer = 'yes'
+      end
+
+      it 'returns the name for the next step' do
+        expect(wizard_step.next_step).to eq(:what_a_level_is_required)
+      end
```

Comment:
no a blocker, but I wonder what you think about removing the before step and making this simply an `it` block. (same as below). I feel like we talked about that a bit as a dev retro, but not sure where we landed. Sometimes, it makes things a bit easier to to read, but I don't feel strongly about it. 
```suggestion
      it 'returns the name for the next step' do
        wizard_step.answer = 'yes'
        expect(wizard_step.next_step).to eq(:what_a_level_is_required)
      end
```

Reply:
Agreed :). Done